dictionaries/InternalTaintSinkMap.php

```diff
@@ -5,10 +5,24 @@
 return [
 'exec' => [['shell']],
 'create_function' => [['text'], ['eval']],
-'file_get_contents' => [['text']],
-'file_put_contents' => [['shell']],
-'fopen' => [['shell']],
-'header' => [['text']],
+'file_get_contents' => [['file']],
+'file_put_contents' => [['file']],
```

Comment:
Technically that could also be used for RCE (by writing a file somewhere in the web root).

I'll make sure to document the different risks in the documentation for the `file` type. In the future we could consider having separate types that all potentially share similar sanitizers though.